src/styles/mixins/_tables.scss

```diff
@@ -0,0 +1,220 @@
+    width: 100%;
+  }
+
+  .table-wrapper {
+    overflow: auto;
+    width: 100%;
+
+    table {
+      border-collapse: collapse;
+      max-width: 100%;
+      width: 100%;
+
+      th,
+      td {
+        @include truncate;
+
+        padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
+        vertical-align: middle;
+        white-space: nowrap;
+      }
+
+      thead {
+        border-bottom: var(--kui-border-width-10, $kui-border-width-10) solid var(--kui-color-border, $kui-color-border);
+        height: 44px;
+        position: sticky;
+        top: 0;
+
+        &.is-scrolled {
+          background-color: var(--kui-color-background, $kui-color-background);
+        }
+
+        tr {
+          position: relative;
+
+          &:after {
+            box-shadow: none;
+            content: "";
+            height: 100%;
+            left: 0;
+            opacity: 0;
+            // Super-important to allow clicking on table rows in Safari.
+            // This allows clicks to pass through the "invisible" :after layer
+            pointer-events: none;
+            position: absolute;
+            transition: opacity $kongponentsTransitionDurTimingFunc;
+            width: 100%;
+            z-index: -1;
+          }
+
+          &.is-scrolled {
+            &:after {
+              border-bottom: var(--kui-border-width-10, $kui-border-width-10) solid
+                var(--kui-color-border, $kui-color-border);
+              box-shadow: var(--kui-shadow, $kui-shadow);
+              opacity: 1;
+              transition: opacity $kongponentsTransitionDurTimingFunc;
+            }
+          }
+
+          th {
+            color: var(--kui-color-text-neutral, $kui-color-text-neutral);
+            font-size: var(--kui-font-size-30, $kui-font-size-30);
+            font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
+            line-height: var(--kui-line-height-30, $kui-line-height-30);
+            padding: $tableThPaddingY var(--kui-space-60, $kui-space-60);
+            text-align: left;
+            vertical-align: bottom;
+
+            &.resizable {
+              // set min width so the column can't be collapsed to nothing - avoiding bad UX
+              min-width: 40px !important; // needs important because resizing will set min-width inline
+              position: relative;
+
+              // when sortable or has tooltip (or both), we need to increase the min-width to account for icons
+              &.sortable,
+              &.has-tooltip {
+                min-width: 80px !important; // needs important because resizing will set min-width inline
+              }
+
+              &.sortable.has-tooltip {
+                min-width: 100px !important; // needs important because resizing will set min-width inline
+              }
+
+              .resize-handle {
+                cursor: col-resize;
+                height: v-bind("headerHeight");
```

Comment:
`headerHeight` is deleted from the source code, so now `.resize-handle` can't be interacted with, which essentially breaks `resizeColumns`:

<img width="1713" alt="image" src="https://github.com/user-attachments/assets/71eb6b3f-70da-42c1-9779-dc9406aaee6e">

Reply:
Good catch, thanks!

Reply:
Pushed a fix